specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/NetworkWatcherMultiEndpointsConnectionMonitorCreate.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Please add reference to this example to `networkWatcher.json`

Reply:
done